features.yaml

```diff
@@ -207,7 +207,7 @@ features:
   - name: "Mutual TLS Migration"
```

Comment:
Is the `Mutual TLS Migration` even considered as a feature? I thought it is just a task documentation about how to use PeerAuthentication strict mode to lock-down mTLS.

Reply:
@yangminzhu It is mentioned as a feature with current status as Beta in the [feature status](https://istio.io/latest/docs/releases/feature-stages/#security-and-policy-enforcement) page for Istio. If it is not, then we need to remove this entry and page corresponding to it. Let me know. Thanks for the review.

Reply:
I feel this is more of a task page than a supported feature, not sure why it's added as supported feature in the first place though.

@incfly @myidpt @liminw any thoughts here?

Reply:
I agree with you @yangminzhu . +1 make more sense to just remove it as a feature if possible.

Reply:
@yangminzhu @incfly I will go ahead and remove this from the features.yaml file and also close this PR so that we do not have a feature page for this. We will have to open another doc PR to update the istio.io feature status page. I can take a look at that also and try to remove it from there also. Thanks for the review.